Extending/Key-Vault/index-v10.md

```diff
@@ -0,0 +1,74 @@
+---
+versionFrom: 10.0.0
+meta.Title: "Azure Key Vault"
+meta.Description: "A guide configuring an Azure Key Vault"
+---
+
+# Configuring an Azure Key Vault
+
+From a security perspective, it's always a good solution to store your application secrets (such as a connection string and others keys) in an Azure Key Vault. This article tells you how to configure your application so that it's ready to use a Key Vault
+
+## Installing the package
+
+Before you begin, you need to install the `Azure.Extensions.AspNetCore.Configuration.Secrets` and the `Azure.Identity` NuGet packages. There are two approaches to installing the packages:
+
+1. Use your favorite IDE and open up the NuGet Package Manager to search and install the packages
+1. Use the command line to install the package
+
+## Installing through command line
+
+Navigate to your project folder, which is the folder that contains your .csproj file. Now use the following 'dotnet add package' command to install the package:
+
+```
+dotnet add package Azure.Extensions.AspNetCore.Configuration.Secrets
+dotnet add package Azure.Identity
+```
+
+## Configuration
+The next step is to add the Azure Key Vault endpoint to the 'appsettings.json' file. 
+
+```json
+{
+  "AzureKeyVaultEndpoint": "https://{your-key-vault-name}.vault.azure.net",
+}
+```
+
+After adding the Key Vault endpoint you have to update the `CreateHostBuilder` method which you can find in the `Program.cs` class. 
```

Comment:
I realize that it is a "class", but I think in the context here, it makes more sense to use "file", as it's a file you need to open and edit, right? 🤔 

```suggestion
After adding the Key Vault endpoint you have to update the `CreateHostBuilder` method which you can find in the `Program.cs` file. 
```